### DISCUSSION TITLE
Kubevela 安装要的从k8s版本问题

### DISCUSSION THREAD

**Discussion by skyshaokai** [kubevela/kubevela#3789] [Q&A] [answered]
Kubevela  v1.3要求k8s 集群是>= v1.19，目前自己使用的k8s集群目前都是v1.18 。想安装Kubevela v1.3 必须要求所有k8s 集群都是 >=v1.19 吗？还是说安装 kubebela 的hubcluster 必须 >= v.1.19 ，但是，被管理的 mangedcluster 可以小于 v1.19 ，例如 v1.18 版本的k8s。

如下图多集群管理：
<img width="875" alt="image" src="https://user-images.githubusercontent.com/11512967/165880867-0a7f691c-ae5b-4f36-ba12-95d2c9062255.png">

**Comment by wonderflow** [comment, marked as answer]
* 作为管控，1.18 目前主要功能均不影响使用，可以尝试一下，集群要求的版本 >=v1.19 主要指我们有测试兼容性。
* 作为被管控集群，完全不受影响
